web_graph_improved/__openerp__.py

```diff
@@ -27,7 +27,7 @@
     'author': "Leonardo Donelli,Odoo Community Association (OCA)",
     'license': 'AGPL-3',
     'website': 'http://learts.glaucus.in',
-    'depends': ['web_graph'],
+    'depends': ['web'],
     'data': ['view/graph_improved_view.xml'],
     'installable': False,
```

Comment:
🤔

Reply:
Hi @mreficent, your review is quite cryptic !
could you be more explicit ? 
regards.

Reply:
It's a bit odd doing changes in a module that is not installable 😏

Reply:
yes ! but even if it's not installable, it is read by odoo, and as it depends on a non existing module, it make the upgrade failing. 
The alternative was to delete fully this module, that has nothing to do in V9 serie. I thought it was less brutal, just to remove the undesired dependency.

Reply:
I think the best solution is just delete this deprecated module from the repo! :)

Reply:
Please, delete it. And also, you could delete it from v10 too :)